lua/autorun/server/cfc_weapon_lockouts_init.lua

```diff
@@ -0,0 +1,40 @@
+local IsValid = IsValid
+local CurTime = CurTime
+local ACT_VM_RELOAD = ACT_VM_RELOAD
+
+local function setLockoutDuration( wep )
+    local reload = wep:SelectWeightedSequence( ACT_VM_RELOAD )
+    wep.CFC_LockoutTime = wep:SequenceDuration( reload )
+end
+
+hook.Add( "WeaponEquip", "CFC_WeaponLockouts", function( wep, ply )
```

Comment:
```suggestion
hook.Add( "WeaponEquip", "CFC_WeaponLockouts_SetLocket", function( wep, ply )
```

Reply:
Does this run when someone gets a swep normally, or only when its picked from ground.

Reply:
When equipped normally too